docs/core/runtime-config/garbage-collector.md

```diff
@@ -709,13 +709,21 @@ Project file:
 
 ## Standalone GC
 
-- Specifies the name of a GC native library that the runtime loads in place of the default GC implementation. This native library needs to reside in the same directory as the .NET runtime (**coreclr.dll** on Windows, **libcoreclr.so** on Linux).
+- Specifies the name of a GC native library that the runtime loads in place of the default GC implementation. This native library needs to reside in the same directory as the assembly contained the `Main` method or as the .NET runtime (**coreclr.dll** on Windows, **libcoreclr.so** on Linux).
 
```

Comment:
could we specify the load order here, if I have a GC dll in both where my assembly is and next to coreclr.dll?